### PR TITLE
Lua 5.1 minimum compatibility

### PR DESCRIPTION
**Pull request by Omikhleia**
See https://github.com/Omikhleia/resilient.sile/pull/70

### CODE REVIEW

File: .luacheckrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-std = "max"
+std = "min+sile"
 include_files = {
   "**/*.lua",
   "sile.in",
```
Comment (alerque) [inline .luacheckrc]:
```suggestion
```
This line looks like copy-paste cruft from SILE that is irrelevant to this project.

File: .luacheckrc [hunk 2]
```diff
@@ -11,23 +11,14 @@ exclude_files = {
   "compare-*",
   "sile-*",
```
Comment (alerque) [inline .luacheckrc]:
```suggestion
```

This line and the two before it (benchmark and compare) are also artifacts from SILE development and not relevant here

File: lua-libraries/qrencode.lua [hunk 3]
```diff
@@ -175,15 +175,14 @@ end
...
-	local mode
 	if string.match(str,"^[0-9]+$") then
 		return 1
 	elseif string.match(str,"^[0-9A-Z $%%*./:+-]+$") then
 		return 2
 	else
 		return 4
 	end
-	assert(false,"never reached")
+	assert(false,"never reached") -- luacheck: ignore
```
Comment (alerque) [inline lua-libraries/qrencode.lua:185]: Is this really a vendored library or something you worked up?

Comment (Omikhleia) [reply lua-libraries/qrencode.lua:185]: Vendored from https://github.com/speedata/luaqrcode (which doesn't have a rockspec).

Just temporarily enabling luacheck on it, and upgrading to their latest version (with linting commits from me, the author etc.)

Comment (alerque) [reply lua-libraries/qrencode.lua:185]: Gotcha. You know you can lint locally for temporary answers too ;-) No problems doing it in CI I guess, just faster to do locally especially for one-off answers to questions outside of the remote configuration.

**Comment by Omikhleia** [comment]
All good.
Updated the vendored qrencode from speeddata - There are still some linting errors, so I left the lua-libraries unchecked, but nothing relates to the Lua min profile.
No change in behavior on my tests.

-> No need for a patch release.

**Pull request merged**